config/routes.rb

```diff
@@ -2,14 +2,20 @@
   root 'homes#index'
   get '/landing', to: 'homes#show'
   get '/workouts/new', to: 'homes#index'
+  get '/workouts/:id/activities/edit', to: 'homes#index'
+  get '/workouts/:id', to: 'homes#index'
+  patch '/api/v1/activities', to: 'api/v1/activities#update'
```

Comment:
if we are routing to a normal controller, we should ideally create said activies controller and prepare routing using the normal `resources` method

Reply:
actually, you might already have this, so maybe this line can be removed?